### PR TITLE
feat: Add CI for prod deployment

### LINKED ISSUE #19: Add CI for upload images to ghcr and add compose for prod server
- [ ] Modify CI
- [ ] Add prod docker compose

### PR DESCRIPTION
**Pull request by coolusaHD**
Closes #19

### CODE REVIEW

File: prod.docker-compose.yml
Comment (longpollehn) [inline prod.docker-compose.yml:1]: #TODO: add image

Comment (longpollehn) [inline prod.docker-compose.yml:1]: #TODO: add image

Comment (ghost) [inline prod.docker-compose.yml:1]: #TODO: add image

Comment (ghost) [reply prod.docker-compose.yml:1]: TODO: Update current compose file with newer compose file on the same branch. 

Or even better: write a script to start only specific services. This avoids code redundancy. See CI pipeline

Comment (coolusaHD) [reply prod.docker-compose.yml:1]: that would make it harder to start 
as the prod compose file really differs from the normal one

Or do I understand you wrong ?

Comment (ghost) [reply prod.docker-compose.yml:1]: I see, the docker image will differs

**Review by ghost — APPROVED** [review]

**Pull request merged**